config/environments/staging.rb

```diff
@@ -0,0 +1,90 @@
+Rails.application.configure do
```

Comment:
@simaob So I added this to the config environments folder so we can differentiate between production and staging. Is this ok? Or do I have to add something else?